### DISCUSSION TITLE
Unreal Engine - JsbSim rocket simulation.

### DISCUSSION THREAD

**Discussion by Zaryob** [JSBSim-Team/jsbsim#905] [Q&A] [answered]
Hi folks. 
I'm curious about simulating rockets. In my case I have F15 model and going to simulate rocket launched from F15. I individually simulate F15 and AIM9 rocket but is there any way to simulate them together?

I previously tried to spawn two plane at the same time, but it was crashed Unreal Editor.

**Comment by gallonmate** [comment, marked as answer]
For now, you'll have to make missiles yourself in UE5. The UE5 plugin currently doesn't support more than one simulation. It's possible someone will add support, but for now it doesn't work.
